tests/ml_models/test_question_answering.py

```diff
@@ -0,0 +1,73 @@
+
+from opensearch_py_ml.ml_models import QuestionAnsweringModel
+
+# Save our model as pt or onnx
+model_id = "distilbert-base-cased-distilled-squad"
+folder_path = "question-model-folder"
+our_pre_trained_model = QuestionAnsweringModel(model_id=model_id, folder_path=folder_path, overwrite=True)
+# zip_file_path = our_pre_trained_model.save_as_pt(model_id=model_id, sentences=["for example providing a small sentence", "we can add multiple sentences"])
+zip_file_path = our_pre_trained_model.save_as_onnx(model_id=model_id)
+
+# List of questions to test
+questions = ["Who was Jim Henson?", "Where do I live?", "What's my name?"]
+contexts = ["Jim Henson was a nice puppet", "My name is Sarah and I live in London", "My name is Clara and I live in Berkeley."]
+
+# Obtain pytorch's official model
+from transformers import DistilBertTokenizer, DistilBertForQuestionAnswering
+import torch
+tokenizer = DistilBertTokenizer.from_pretrained('distilbert-base-cased-distilled-squad')
+official_model = DistilBertForQuestionAnswering.from_pretrained('distilbert-base-cased-distilled-squad')
+
+def official_model_answer(question, context):
+    inputs = tokenizer(question, context, return_tensors="pt")
+    with torch.no_grad():
+        outputs = official_model(**inputs)
+    answer_start_index = torch.argmax(outputs.start_logits, dim=-1).item()
+    answer_end_index = torch.argmax(outputs.end_logits, dim=-1).item()
+    predict_answer_tokens = inputs['input_ids'][0, answer_start_index : answer_end_index + 1]
+    official_answer = tokenizer.decode(predict_answer_tokens)
+    return official_answer
+
+def test_onnx():
```

Comment:
what are we asserting in the test?

Reply:
We should assert if our answer string is equal to the official answer. I can add an assertion statement instead of printing the two outputs!

Reply:
A question here. Would response be always same?

Reply:
I think generative models have randomness. Here in the three runs and the outputs were the same.

Reply:
One way is to just calculate the cosine similarity between expected output and the actual output. Usually they don't differ much and it will be having high value.